index.es6

```diff
@@ -19,6 +25,11 @@ export default function createMiddleware(dsn, cfg={}) {
   }
 
   return store => next => action => {
+    const {
+      actionTransformer = identity,
+      stateTransformer = identity,
+      logger = console.error
```

Comment:
This should be `console.error.bind(console)`. Some browser implementations of `console` differs.

<img width="333" alt="screenshot 2016-09-26 11 57 59" src="https://cloud.githubusercontent.com/assets/161365/18830394/cb9954e0-83e0-11e6-9a54-954566e61634.png">